packages/gatsby-plugin-image/src/babel-plugin-parse-static-images.ts

```diff
@@ -5,7 +5,6 @@ import fs from "fs-extra"
 import path from "path"
 import { slash } from "gatsby-core-utils"
 
-import template from "@babel/template"
```

Comment:
I'd change all the babel imports above to:

```typescript
import type {
  PluginObj,
  template as BabelTemplate,
  types as BabelTypes,
} from "@babel/core"
```

...then change below

Reply:
Works for me!